autodispose-rxlifecycle/src/main/java/com/ubercab/autodispose/rxlifecycle/RxLifecycleInterop.java

```diff
@@ -28,7 +28,7 @@ private RxLifecycleInterop() {
 
   private static final Object DEFAULT_THROWAWAY_OBJECT = new Object();
 
-  public static <E> ScopeProvider fromLifecycle(final LifecycleProvider<E> provider) {
+  public static <E> ScopeProvider from(final LifecycleProvider<E> provider) {
```

Comment:
Missed this in my earlier comments, but these two methods should each have docs

Reply:
Fixed in https://github.com/uber/AutoDispose/pull/118/commits/c2e63ffab40293fb7c847b8ede3d2d8aa7c93976